pkg/controller/scyllaoperatorconfig/controller.go

```diff
@@ -185,6 +193,28 @@ func (opc *Controller) Run(ctx context.Context, workers int) {
 		}()
 	}
 
+	opc.wg.Add(1)
+	go func() {
+		defer opc.wg.Done()
+		wait.UntilWithContext(ctx, func(ctx context.Context) {
+			klog.V(4).InfoS("Periodically enqueuing %q ScyllaOperatorConfig", naming.SingletonName)
+
+			soc, err := opc.scyllaOperatorConfigLister.Get(naming.SingletonName)
```

Comment:
this may technically be not found but I don't think you need to list at all
````golang
	key, err := h.keyFunc(metav1.ObjectMeta{
		Namespace: "",
		Name:      naming.SingletonName,
	})
```